src/app/routes/index.js

```diff
@@ -13,10 +19,23 @@ import {
   radioAndTvPath,
 } from './regex';
 
+// CPS Asset Mapping to PageType
 const CpsAsset = props => {
-  const type = pathOr('STY', ['pageData', 'metadata', 'type'], props);
-  const Page = type === 'FIX' ? FrontPage : CpsAssetPage;
-  return Page({ ...props, pageType: type });
+  const type = path(['pageData', 'metadata', 'type'], props);
+
```

Comment:
I thought the logging was quite useful - I had actually added this in another PR (#4967) but it would be much simpler here?

Reply:
I like the more centralised approach your PR is taking for the logging. I wonder if react router accepts some form of middleware or something similar we can hook into for logging purposes